data/conf/infrasim.yml

```diff
@@ -0,0 +1,60 @@
+---
+#This file is used as InfraSIM configuration file
+
+main:
+    #Supported vnode names:
+    #	quanta_d51
+    #	quanta_t41
+    #	dell_c6320
+    #	dell_r630
+    #	s2600kp
+    #	s2600tp
+    #	s2600wtt
+    node: quanta_d51

```

Comment:
This PR is the first step of my refactor to achieve **compute** this sprint.

[->] Update existing .conf to .yml and corresponding function and test
[ ] Introduce device class in tree and update .yml to the template
[ ] Enhance device customization features, add unit test

After these three steps, we can use infrasim-vxrail style to compose qemu command. 
The next step (sprint) is to introduce infrasim-vxrail style's emu_file and smbios definition, so we can finish **compute** implementation.
Besides **compute** implementation in infrasim-vxrail style, we would also have to introduce how infrasim-vxrail compose **compute** to **chassis**, then we shall consider node3/node4's differentiation.